docs/docsite/rst/galaxy/user_guide.rst

```diff
@@ -270,6 +270,9 @@ Use the following example as a guide for specifying roles in *requirements.yml*:
     # from galaxy
     - name: yatesr.timezone
 
+    # from locally cloned git repository, relative paths are not supported
```

Comment:
Can you change this comment to say something about relative paths are not supported with the `src`: option (or File:/// whichever is the one causing it?

Reply:
looks ok now? Also avoided the negation ;)